bin/windows-toolset.ps1

```diff
@@ -40,22 +40,27 @@ if (!(Get-Command choco -ErrorAction SilentlyContinue)) {
     & $scriptDirectory\install-chocolatey.ps1
 }
 
-Install-Choco 7zip.commandline 9.20.0.20150210
+Install-Choco 7zip.commandline 9.20.0.20150210 'http://nexus.delivery.puppetlabs.net/service/local/nuget/temp-build-tools/'
```

Comment:
Since it's reused a bunch, and it's such a long URL, could we do:

```powershell
$tempTools = 'http://nexus.delivery.puppetlabs.net/service/local/nuget/temp-build-tools/'

Install-Choco 7zip.commandline 9.20.0.20150210 -Source $tempTools
Install-Choco cmake 3.2.2 -Source $tempTools
Install-Choco git.install 2.6.2.20151028 -Source $tempTools
Install-Choco Wix35 $Wix35_VERSION -Source $tempTools
```